website/content/commands/debug.mdx

```diff
@@ -71,14 +71,14 @@ all targets for 2 minutes.
-| `cluster` | A list of all the WAN and LAN members in the cluster.                                                                                                                               |
-| `metrics` | Metrics from the in-memory metrics endpoint in the target, captured at the interval.                                                                                                |
-| `logs`    | `DEBUG` level logs for the target agent, captured for the interval.                                                                                                                 |
-| `pprof`   | Golang heap, CPU, goroutine, and trace profiling. This information is not retrieved unless [`enable_debug`](/docs/agent/options#enable_debug) is set to `true` on the target agent. |
+| Target    | Description                                                                                                                                                                                                                                                                           |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `agent`   | Version and configuration information about the agent.                                                                                                                                                                                                                                |
+| `host`    | Information about resources on the host running the target agent such as CPU, memory, and disk.                                                                                                                                                                                       |
+| `cluster` | A list of all the WAN and LAN members in the cluster.                                                                                                                                                                                                                                 |
+| `metrics` | Metrics from the in-memory metrics endpoint in the target, captured at the interval.                                                                                                                                                                                                  |
+| `logs`    | `DEBUG` level logs for the target agent, captured for the interval.                                                                                                                                                                                                                   |
```

Comment:
I think this should say "captured for the whole duration" now right?

Reply:
> I think this looks good.
> 
> Not sure if other reviewers want to take another pass - might be worth a ping to see but I'll approve this 😄 .
> 
> There was one integration test failure that looks like a flake and I just re-ran it. If it fails again it might be worth double-checking but I expect it's unrelated as I don't think we even run this command in envoy integration tests anywhere.

I will check if @dnephin or @freddygv  can take a quick look and merge it. 
Thank you 😄 !!